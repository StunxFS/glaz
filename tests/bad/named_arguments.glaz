fn x(a: i32) {}

fn main() {
  x(a=10, b=1);
}
