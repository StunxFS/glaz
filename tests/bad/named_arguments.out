tests/bad/named_arguments.glaz:4:6: error: argument `a` is not optional
     | 
   4 |   x(a=10, b=1);
     |     ^^^^
tests/bad/named_arguments.glaz:4:12: error: function `x` does not have an argument called `b`
     | 
   4 |   x(a=10, b=1);
     |           ^^^
tests/bad/named_arguments.glaz:1:7: warning: argument `a` declared but never used
     | 
   1 | fn x(a: i32) {}
     |      ^^^^^^
     = help: if this is intentional, prefix it with an underscore: `_a`
     = help: or use the blank identifier (`_`)
glaz error: could not compile `named_arguments`, aborting due to 2 previous errors.
